utils/publish/pre-publish-check.js

```diff
@@ -1,76 +0,0 @@
-/* eslint-disable no-console */
-/* eslint-disable import/no-commonjs */
-
-/**
- * Pre-publish checks to verify that our publish will go smoothly.
- * This file is used by the npm publish script (manual publish).
- */
-const path = require("path");
-const {exec} = require("child_process");
-const glob = require("glob");
-
-const inquirer = require("inquirer");
-const {
-    checkPublishConfig,
-    checkPackageMain,
-    checkPackageModule,
-    checkPackageSource,
-    checkPackagePrivate,
-} = require("./pre-publish-utils.js");
-
-const checkNpmUser = (currentUser) => {
-    if (currentUser.trim() !== "khanacademy") {
-        console.error(
-            `ERROR: You are not logged in to NPM as "khanacademy". ` +
-                `Run "npm login" and use the password from: ` +
-                `https://phabricator.khanacademy.org/K207`,
-        );
-        process.exit(1);
-    }
-};
-
-glob(
-    path.join(__dirname, "..", "..", "packages", "**", "package.json"),
-    (err, pkgPaths) => {
-        let warnings = false;
-
-        for (const pkgPath of pkgPaths) {
-            const pkgJson = require(path.relative(__dirname, pkgPath));
-
-            checkPublishConfig(pkgJson);
-            checkPackageMain(pkgJson);
-            checkPackageModule(pkgJson);
-            checkPackageSource(pkgJson);
-            warnings = checkPackagePrivate(pkgJson);
```

Comment:
Does our `changeset` setup do these checks for us?

Reply:
Kinda... we have a separate `pre-publish-check-ci.js` file that was created for that case, but I probably forgot to remove this non-ci file (that was the previous way we used to do releases).